.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: php
 php:
-  - 5.4
+  - 5.5
   - 5.6
   - 7.0
```

Comment:
My dev machines are php 5.6. Most linux out of the box is something lower than 7. Besides I find that things that work across releases tend to be more stable.